OMEdit/OMEdit/OMEditGUI/Options/OptionsDialog.cpp

```diff
@@ -1680,6 +1684,20 @@ void OptionsDialog::reject()
   QDialog::reject();
 }
 
+void OptionsDialog::reset()
+{
+  const QString title = "Reset to default";
+  const QString text = "Are you sure that you want to reset OMEdit? This operation can not be undone. Please back up your settings file before proceeding, restart to have the changes take effect.";
+  readSettings();
```

Comment:
Why do you have to read here?

Reply:
Artifact that I forgot to remove.. will take it away